package.json

```diff
@@ -12,17 +17,19 @@
     "git-pull": "*",
     "quarantine": "*",
     "redwrap": "*",
-    "request": "*",
+    "request": "2.54.0",
```

Comment:
This is to attempt to fix the timeout issue with later versions of node, which causes Kassy to crash silently.
See https://github.com/request/request/issues/1919